src/com/linkedin/parseq/TaskDescriptor.java

```diff
@@ -0,0 +1,20 @@
+package com.linkedin.parseq;
+
+/**
+ * An API to provide description for task.
+ *
+ * parseq-lambda-names provide an implementation for this interface using Java SPI. Any changes made to this interface
+ * might require a change to implementation in parseq-lambda-names.
+ *
+ * @author Siddharth Sodhani (ssodhani@linkedin.com)
+ */
+public interface TaskDescriptor {
+
+  /**
+   * Give class name which could correspond to generated lambda expressions etc infer appropriate description for it
```

Comment:
What if lambda class was not found e.g. if I pass a gibberish to this function? What is it supposed to return?
Perhaps it would be better to change signature to `Optional<String> getDescription(String className);`?